engine/lib/generic_printer/generic_printer_template.ml

```diff
@@ -318,6 +319,16 @@ struct
       method projection_predicate ~impl:_ ~assoc_item:_ ~typ:_ =
         default_document_for "projection_predicate"
 
+      method quote_content_Expr _x1 = default_document_for "quote_content_Expr"
+
+      method quote_content_Pattern _x1 =
+        default_document_for "quote_content_Pattern"
+
+      method quote_content_Typ _x1 = default_document_for "quote_content_Typ"
+
+      method quote_content_Verbatim _x1 =
+        default_document_for "quote_content_Verbatim"
+
```

Comment:
Related to my [other comment in `coq/coq_backend.ml`](https://github.com/hacspec/hax/pull/1206/files#r1903775169):

Those will disappear after implementing default methods in generic_printer.ml, don't forget to regenerate at that point!